name.abuchen.portfolio/src/name/abuchen/portfolio/datatransfer/pdf/DkbPDFExtractor.java

```diff
@@ -268,6 +268,11 @@ private void addDividendeTransaction()
                 .match("^(?<isin>[\\w]{12}) \\((?<wkn>.*)\\)$")
                 .match("^Ertrag pro St. [\\.,\\d]+ (?<currency>[\\w]{3})$")
                 .assign((t, v) -> t.setSecurity(getOrCreateSecurity(v)))
+                
+                // Ex-Dividenden-Tag 29.11.2016
```

Comment:
// Ex-Dividenden-Tag 29.11.2016
not the same for regEx
`^Ex-Tag (?<exDate>\\d+.\\d+.\\d{4}).*`
Is `.*` really needed?

Reply:
as we have multiple examples at testcases I changed it to 

`// Ex-Tag 29.11.2016
                // Ex-Tag 30.11.2017 Herkunftsland Irland
                // Ex-Tag 27.12.2018 fonds) 0,148099700 USD
                // Ex-Tag 29.01.2015 Ausländischer Dividenden- und Veräußerungs-
                .section("exDate").optional()
                .match("^Ex-Tag (?<exDate>\\d+.\\d+.\\d{4}).*")                
                .assign((t,v) -> t.setExDate(asDate(v.get("exDate")).toLocalDate()))
                
                // Ex-Tag 09.02.2017 Art der Dividende Quartalsdividende
                .section("exDate", "note").optional()
                .match("^Ex-Tag (?<exDate>\\d+.\\d+.\\d{4}) Art der Dividende (?<note>.*)$")                
                .assign((t,v) -> {
                    t.setExDate(asDate(v.get("exDate")).toLocalDate());
                    t.setNote(v.get("note").trim());
                })`

Reply:
as we have multiple examples in test case I changed it into 2 parts

// Ex-Tag 09.02.2017 Art der Dividende Quartalsdividende
                .section("exDate", "note").optional()
                .match("^Ex-Tag (?<exDate>\\d+.\\d+.\\d{4}) Art der Dividende (?<note>.*)$")                
                .assign((t,v) -> {
                    t.setExDate(asDate(v.get("exDate")).toLocalDate());
                    t.setNote(v.get("note").trim());
                })